components/bl-gallery-component/src/components/gallery.js

```diff
@@ -0,0 +1,51 @@
+import LightBox from '../lib/lightbox-jquery.js';
+import { GalleryImage } from './gallery-image.js';
+
+export function Gallery({ images, options }) {
+  const {
+    alwaysShowNavOnTouchDevices,
+    imageLabel,
+    separator,
+    disableScrolling,
+    fadeDuration,
+    imageFadeDuration,
+    positionFromTop,
+    resizeDuration,
+    showImageCount,
+    wrapAround,
+  } = options;
+
+  LightBox.option({
+    'alwaysShowNavOnTouchDevices': alwaysShowNavOnTouchDevices,
+    'albumLabel': `${imageLabel} %1 ${separator} %2`,
+    'disableScrolling': disableScrolling,
+    'fadeDuration': fadeDuration,
+    'imageFadeDuration': imageFadeDuration,
+    'positionFromTop': positionFromTop,
+    'resizeDuration': resizeDuration,
+    'showImageNumberLabel': showImageCount,
+    'wrapAround': wrapAround,
+  });
+
+  return (
+    <div className="bl-customComponent-lightBox">
+      {
+        images.map(image => {
+          const { title, url, height } = image;
+          const fixedTitle = title.replace(/</g, "&lt;").replace(/>/g, "&gt;");
```

Comment:
Create an utility function, and move outside of component

fixedTitle => escapedTitle